### ISSUE TITLE
Error loading syntax file

### ISSUE THREAD

**Issue by SonGokussj4** [jfcherng/Sublime-ShellScriptImproved#6]
Hello, been using your Improved shell for a couple of days now :-) Good work.

When sublime updated to the new version though, ShellScript Improved stopped to work.
It's like a 1-3 days back. I've seen it today for the first time. Collegue on Monday.

```
Error loading syntax file "Packages/ShellScriptImproved/Shell-Unix-Generic.sublime-syntax": Error in regex: empty range in char class in regex \+{1,2}|-{1,2}|!|~|\*{1,2}|/|%|([=!*/%+-&^|]|<<?|>>?)?=|^|\|{1,2}|&{1,2}|[?:,<>]
```

**Comment by joaociocca** [comment]
I just started using Sublime and SSI, so I'm at a disadvantage 'cause I've never seen SSI at work... subscribing here to keep updated when a fix comes out! =)

**Comment by jfcherng** [comment]
I guess there are some changes in ST's regex engine. This can be reproduced in 3103 but 3110 works fine. It should be fixed in 1.0.21.
